build_tfm.py

```diff
@@ -454,11 +442,41 @@ def _copy_library(source, toolchain):
     if not isdir(regression_lib_dest):
         os.makedirs(regression_lib_dest)
     shutil.copy2(
-        join(regression_lib_src, "libtfm_non_secure_tests.a"),
+        join(regression_lib_src, dest_lib_name),
```

Comment:
Could you add logging here as well so we see where the library is copied?

Reply:
This is copied in tfm regression folder... this is internally maintained